src/PhpSpec/Formatter/ProgressFormatter.php

```diff
@@ -87,7 +87,7 @@ function ($count) use ($total) {
     private function getBarLengths($percents)
     {
         $stats = $this->getStatisticsCollector();
-        $specProgress = ($stats->getTotalSpecs()+1)/$stats->getTotalSpecsCount();
+        $specProgress = $stats->getTotalSpecs() ? ($stats->getTotalSpecs()+1)/$stats->getTotalSpecsCount() : 0;
```

Comment:
@ciaranmcnulty the division by 0 happens when ``getTotalSpecsCount`` returns 0. So the safeguard should check this method too. The name of the method is not involved here. Your logic is